README.md

```diff
@@ -17,6 +17,26 @@ This is currently work-in-progress.
 - Input the bot token [(guide)](https://discordjs.guide/preparations/setting-up-a-bot-application.html)
 - Run `./ukulele` to build and run the application
 
+### Using Docker
+#### Requirements
+- Docker (Engine: 18.06.0+)
+- Docker-Compose
+
+#### Running
+```shell script
+# Create DB directory and own it to 999
+mkdir db && chown -R 999 db/
```

Comment:
Is this step necessary? I was under the impression that docker-compose would create the directory automatically

Reply:
It is neccessary in this case, as i do not run ukulele as root but the ukulele user with uid 999.
If you do not create the folder and own it before hand docker *will* create it for you but since docker is running as root on the host the folder will be owned by root and mount it with rood uid/gid preventing the app to write / read